modules/apps/portal-security-audit/portal-security-audit-event-generators-api/src/main/java/com/liferay/portal/security/audit/event/generators/constants/EventTypes.java

```diff
@@ -32,6 +32,8 @@ public interface EventTypes {
 
 	public static final String LOGIN = "LOGIN";
 
+	public static final String LOGIN_DOES_NOT_EXIST = "LOGIN_DOES_NOT_EXIST";
+
```

Comment:
I think we can call this LOGIN_DNE because "DNE" is a code that is being used elsewhere

Reply:
This was requested on the forwarded PR: [comment](https://github.com/brianchandotcom/liferay-portal/pull/135456#issuecomment-1568235768)

Reply:
I see, thanks for highlighting.
I think though that BChan made that suggestion without seeing the full context. If you highlight https://github.com/liferay/liferay-portal/blob/50888548989ba65c75f7cb7c9c90fa1957cd2117/portal-kernel/src/com/liferay/portal/kernel/security/auth/Authenticator.java#L24 to him then he will probably prefer `LOGIN_DNE`.

Reply:
Alright, I will reference that class then!